cg/apps/demultiplex/sample_sheet/sample_sheet_creator.py

```diff
@@ -76,7 +77,13 @@ def add_override_cycles_to_samples(self) -> None:
         raise NotImplementedError("Impossible to add override cycles to samples from parent class")
 
     def remove_unwanted_samples(self) -> None:
-        """Filter out samples with single indexes."""
+        """Filter out samples with single indexes if applicable."""
+        if self.flow_cell.sequencer_type in [Sequencers.HISEQGA, Sequencers.HISEQX]:
+            LOG.debug(
+                "Skipped removal of single-indexed samples for"
+                f"{self.flow_cell.sequencer_type} flow cell"
+            )
+            return
```

Comment:
You are right, we can use both index cycles and override cycles to set up the configuration for single and double indexes. However, this can be done in the V2 sample sheet but not in the V1. I am not done deprecating the V1 yet, so I thought this solution could work in the meantime, as this PR only adds the new RunParameters parsing (otherwise it would be a PR hard to review). Should I leave this as it is in this PR?  As it is, it does not change anything for the sample sheet generation of NovaSeq FCs.

Reply:
I will resolve this and implement this in the following PR

Reply:
Rolling back to the original, this will be implemented later